modules/hello-world/pages/sample-application.adoc

```diff
@@ -5,69 +5,77 @@
 :page-topic-type: tutorial
 :page-aliases: ROOT:sample-application,ROOT:tutorial4,ROOT:sample-app-backend
 
+:travel-sample-git-project: try-cb-nodejs
+:travel-sample-entrypoint: index.js
+
+// TODO: move these to attributes.adoc
+:name-sdk: Node.js SDK
+:version-sdk: 3.0
+
 [abstract]
 {description}
-This version also demonstrated the Developer Preview of the upcoming Collections feature.
+This version demonstrates various features of Couchbase, including the Collections feature new in Server 7.0.
 
-include::6.5@sdk:shared:partial$sample-application.adoc[tag=prereq]
 
+== Quick Start
 
-== Preparation
+Fetch the https://github.com/couchbaselabs/{travel-sample-git-project}[Couchbase {name-sdk} travel-sample Application REST Backend] from github:
 
-As well as the xref:start-using-sdk.adoc[Node.js SDK 3.0] and Couchbase Server, 
-set up as described above, you will need `git` to fetch the travel sample application code:
-
-[source,bash]
+[source,bash,subs="+attributes"]
```

Comment:
Needs to be console,
like in the other docs - 
bash highlighting is geared towards .sh scripts  ;)